application/src/main/java/org/opentripplanner/transit/service/TransitService.java

```diff
@@ -320,4 +320,12 @@ List<TripTimeOnDate> stopTimesForPatternAtStop(
    * @return - A list of TripOnServiceDates
    */
   List<TripOnServiceDate> getTripOnServiceDates(TripOnServiceDateRequest request);
+
+  /**
+   * Checks if a trip with the given ID exists in the model.
+   *
+   * @param id the ID of the trip to check
```

Comment:
Prefer {@link FeedScopedId}

Reply:
Fixed in f40626194bfc67e2bfdac64f41eb2d69d881d397